test/fixtures/gcs-fixture/src/main/java/fixture/gcs/GoogleCloudStorageHttpHandler.java

```diff
@@ -237,14 +237,16 @@ public void handle(final HttpExchange exchange) throws IOException {
                 BytesReference blob = blobs.get(blobName);
                 final String range = exchange.getRequestHeaders().getFirst("Content-Range");
                 final Integer limit = getContentRangeLimit(range);
-                final int start = getContentRangeStart(range);
-                final int end = getContentRangeEnd(range);
 
                 blob = CompositeBytesReference.of(blob, requestBody);
                 blobs.put(blobName, blob);
 
                 if (limit == null) {
-                    exchange.getResponseHeaders().add("Range", String.format(Locale.ROOT, "bytes=%d/%d", start, end));
+                    if ("bytes */*".equals(range) == false) {
```

Comment:
The new SDK sends the unknown range header to determine where to continue uploading now so we have to deal with that :) For now I just made us not sent a range back but a follow-up would be helpful here to properly account for the bytes we've already seen so that the existing entry from `blobs` is reflected in the response correctly.